.github/workflows/tests.yml

```diff
@@ -264,6 +263,8 @@ jobs:
       - name: Set up Homebrew
         id: set-up-homebrew
         uses: Homebrew/actions/setup-homebrew@master
+        env:
+          HOMEBREW_NO_INSTALL_FROM_API: 1
```

Comment:
This one can be removed if `brew tests` later works in API mode. Would be beneficial to do so as it takes 3-4 minutes to clone Homebrew/core.

The other one for tap_syntax is permanent.

Reply:
@Bo98 Agreed, that'd be great.